edx_lint/tamper_evident.py

```diff
@@ -34,6 +34,10 @@ def write(self, text, hashline=b"# {}"):
                 to the file, with "{}" replaced with the hash.
 
         """
+        # Ensure we are working with bytes.
+        if isinstance(text, str):
+            text = str.encode(text)
```

Comment:
This seems wrong for py2, where we expect a "str", and the .encode will do the wrong thing for non-ascii.  The argument is documented as a bytestring, so why do we need these lines?